webpack/scenes/Subscriptions/UpstreamSubscriptions/UpstreamSubscriptionsTableSchema.js

```diff
@@ -32,7 +31,7 @@ export const columns = (controller, selectionController) => [
       formatters: [
         (value, { rowData }) => (
           <td>
-            <a href={urlBuilder('subscriptions', '', rowData.id)}>
+            <a href={`https://access.redhat.com/management/subscriptions/${rowData.subscription_id}`}>
```

Comment:
I like it a lot. I think the last thing to do is make this open in a new window

Reply:
@jturel Yes. That sounds better. Pushed the changes.